assets/component-deferred-media.css

```diff
@@ -6,6 +6,12 @@
   padding: 0;
   height: 100%;
   width: 100%;
+  border-radius: calc(var(--media-radius) - var(--media-border-width));
+  overflow: hidden;
+}
+
+.global-media-settings--full-width .deferred-media__poster {
+  border-radius: 0;
```

Comment:
Now that `Collage` cards are using settings from `Cards`, the `Media` is still causing changes to the image radius in these blocks:
![image](https://user-images.githubusercontent.com/37168033/149174185-d329a6be-3a91-4819-8e9f-60feeefe4a2d.png)

Should this be replaced with:
```suggestion
.collage-card .deferred-media__poster {
  border-radius: 0;
```
or is this snippet accounting for something else? There's a `.collage-card .deferred-media__poster` selector in `collage.css` already if you prefer to add it there.

Reply:
^ This was actually still an issue on 1124, so I did fix it there.